package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "08-vanilla_rest_api",
+  "version": "1.0.0",
+  "description": "![CF](https://camo.githubusercontent.com/70edab54bba80edb7493cad3135e9606781cbb6b/687474703a2f2f692e696d6775722e636f6d2f377635415363382e706e67) Lab 08: In-Memory Resource API ===",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jtwalters25/08-vanilla_rest_api.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/jtwalters25/08-vanilla_rest_api/issues"
+  },
+  "homepage": "https://github.com/jtwalters25/08-vanilla_rest_api#readme",
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "mocha": "^3.2.0",
+    "node-uuid": "^1.4.7",
+    "superagent": "^3.4.4"
+  },
+  "dependencies": {
+    "chai": "^3.5.0",
+    "node-uuid": "^1.4.7",
+    "superagent": "^3.4.4"
```

Comment:
Please add gulp-mocha and gulp-eslint as devDependencies.  They need to be in your package.json because you are using them for testing.